### DISCUSSION TITLE
Migrating From bumpversion

### DISCUSSION THREAD

**Discussion by jgarte** [callowayproject/bump-my-version#273] [Q&A] [answered]
Hi,

Do you have any recommendations for how to migrate from [bumpversion](https://github.com/peritus/bumpversion)?

**Comment by coordt** [comment, marked as answer]
The key is the configuration. Most of the configuration should be backward-compatible. 

Start by seeing if it recognizes the configuration. Run `bump-my-version show`.

Then run `bump-my-version show-bump` to see if the version progression is what you expect.

Then run `bump-my-version bump <version component> -vv --dry-run` to see what it would do when it bumps your version.

**Reply by jgarte** [reply]
Thanks for the advice. I'll give it a try soon on this project called papermill that is still using bumpversion:

https://github.com/nteract/papermill/issues/814
